doc/tools/pkcs11-tool.1.xml

```diff
@@ -412,7 +412,9 @@
 					<listitem><para>Specify how many bytes of salt should
 					be used in RSA-PSS signatures. Accepts two special values:
 					"-1" means salt length equals to digest length,
-					"-2" means use maximum permissible length.
+					"-2" or "-3" means use maximum permissible length.
+					For verify operation "-2" means that the salt length is autorecovered thom signature.
+					This is supported for opensc pkcs#11 module only.
```

Comment:
This should clarify that it applies only for the -2 with verification. Other special values should work with any module